Apps/CocoaPods-FCM/src/Extensions/StringExtensions.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+extension String {
+    static var abcLetters: String {
+        "abcdefghijklmnopqrstuvwxyz"
+    }
+
+    static func random(length: Int = 10) -> String {
```

Comment:
Should we expose the method in SDK? if all the sample libraries are going to use it? we can probably have some kind of warning on it for not to be used in production.